src/store/reducers.js

```diff
@@ -67,7 +67,7 @@ function entityLoaded(state) {
                 name: 'patch',
                 component: PatchTab
             },
-            (!insights.chrome.isProd || (insights.chrome.isProd && insights?.chrome?.isBeta())) && {
+            {
                 title: 'Resource Optimization',
                 name: 'ros',
                 isVisible: false,
```

Comment:
@Hyperkid123 just curious if we also need to remove `isVisible`? just observed that its not part of other services so got curious.

Reply:
The visibility of the ROS tab is also based on additional inventory data. Some systems might not support the ROS features therefore the tab is hidden. And it is hidden by default because we first have to get some general system data from API. If the condition is met, the flag is flupped.

Reply:
ohh make sense, I got it 💯 
Thanks for clarifying @Hyperkid123 👍🏻